src/Adapters/Laravel/PhikiServiceProvider.php

```diff
@@ -22,6 +22,8 @@ public function register(): void
      */
     public function boot(): void
     {
-        Blade::componentNamespace('Phiki\\Adapters\\Laravel\\Components', 'phiki');
+        if ($this->app->bound('blade.compiler')) {
+            Blade::componentNamespace('Phiki\\Adapters\\Laravel\\Components', 'phiki');
+        }
```

Comment:
Another alternative is to use `Facade::resolved()` helper.

```suggestion
        Blade::resolved(function ($blade) {
            $blade->componentNamespace('Phiki\\Adapters\\Laravel\\Components', 'phiki');
        });
```

Reply:
Thanks @crynobone